### ISSUE TITLE
List: Screen readers are unable to announce the single selection case

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents-react#5272]
### Accessibility Issue with Screen Readers:

Screen readers are unable to announce the single selection case. In contrast, for multi-selection, screen readers successfully convey the MultiSelection message to sightless users.

### Isolated Example

https://m4kpxp.csb.app/

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

Both single selection or multiple selection should be recognized by screen readers.

### Screenshots or Videos

<img width="547" alt="image" src="https://github.com/SAP/ui5-webcomponents-react/assets/5724592/96c1a20c-16e2-430c-aa20-5eec25042896">

### UI5 Web Components for React Version

~1.22.0

### UI5 Web Components Version

~1.19.0

### Browser

Chrome

### Additional Context

I am from SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
